test/val/val_extensions_test.cpp

```diff
@@ -228,4 +228,50 @@ TEST_P(ValidateAMDShaderBallotCapabilities, ExpectFailure) {
 INSTANTIATE_TEST_CASE_P(ExpectFailure, ValidateAMDShaderBallotCapabilities,
                         ValuesIn(AMDShaderBallotGroupInstructions()));
 
+struct ExtIntoCoreCase {
+  const char* cap;
```

Comment:
Should have some cases with Vulkan 1.0 and Vulkan 1.1.

Reply:
Added tests for Vulkan target env.